src/bindings/python/src/pyopenvino/CMakeLists.txt

```diff
@@ -64,8 +61,6 @@ if(OpenVINO_SOURCE_DIR OR InferenceEngineDeveloperPackage_FOUND)
     install(TARGETS ${PROJECT_NAME}
             DESTINATION python/${PYTHON_VERSION}/openvino
             COMPONENT pyopenvino_${PYTHON_VERSION})
-
-    ie_cpack(pyopenvino_${PYTHON_VERSION})
```

Comment:
if it removes cpack generation for this compoent please revert this line. This cpack is used for conda packages.

Reply:
`ie_cpack` should be called once per project. We already have `ie_cpack` call in python bindings root folder https://github.com/openvinotoolkit/openvino/blob/96e345d81ed1564ba39f1ed2ece31a5f0ab6bcfc/src/bindings/python/CMakeLists.txt

Reply:
I suppose in conda build you call cmake on top-level src/bindings/python/CMakeLists.txt file?